modules/installation-mirror-repository.adoc

```diff
@@ -33,20 +33,35 @@ x509: certificate relies on legacy Common Name field, use SANs or temporarily en
 
 Complete the following steps on the mirror host:
 
+[NOTE]
+====
+Optionally, you can use image tag name instead of image digest in the following commands.
+====
+
 . Review the
 link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
-to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
+to determine the version of {product-title} that you want to install.
+The image release advisory includes the pullspec repository and digest.
```

Comment:
Is there any page to allow user to find out the specific image's digest?

Reply:
https://access.redhat.com/errata/product/290/ver=4.5/rhel---8/x86_64/RHBA-2020:3719[the 4.5.11 Bug Fix Advisory] uses `quay.io/openshift-release-dev/ocp-release` for the pullspec repository and  `sha256:4d048ae1274d11c49f9b7e70713a072315431598b2ddbb512aee4027c422fe3e` for the x86-64 4.5.11 release image digest.

This url is give as an example. But this example would be specific to 4.5. This will have to be kept updated. Instead we can redirect to the main errata page where they can voluntarily choose the bug advisory - https://access.redhat.com/downloads/content/290/ver=4.9/rhel---8/4.9.5/x86_64/product-errata

WDYT?

Reply:
> https://access.redhat.com/downloads/content/290/ver=4.9/rhel---8/4.9.5/x86_64/product-errata
+1, but from the url string, I think the link is varying version to version.

Reply:
> > https://access.redhat.com/downloads/content/290/ver=4.9/rhel---8/4.9.5/x86_64/product-errata
> 
> +1, but from the url string, I think the link is varying version to version.

My bad. This is the url I was referring to https://access.redhat.com/downloads/content/290/ver=4.9/rhel---8/. Here I can replace version with a current version placeholder.

Reply:
https://access.redhat.com/downloads/content/290/ver=4.9/rhel---8/ sounds good to me, then also point user to navigate to "Errata" Tab on that page.